web3deployment/compileContracts.js

```diff
@@ -176,4 +185,4 @@ async function main() {
     let v4Output = await compileContracts("sol4");
     let v5Output = await await compileContracts("sol5");
     return [v4Output, v5Output];
-}
\ No newline at end of file
+}
```

Comment:
1 thing missing.

add in file :

main();

so can just run

node compileContracts.js